.travis.yml

```diff
@@ -16,10 +16,22 @@ virtualenv:
   system_site_packages: true
 install:
   - pip install -r https-everywhere-checker/requirements.txt
-before_script:
-  - sh -e /etc/init.d/xvfb start
+  - pip install -r test/chromium/requirements.txt
 env:
   - DISPLAY=':99.0'
+before_script:
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+  - wget https://chromedriver.storage.googleapis.com/2.18/chromedriver_linux64.zip
+  - unzip chromedriver_linux64.zip
+  - sudo mv chromedriver /usr/bin/chromedriver
+  - sudo apt-get install -y python-software-properties
+  - sudo apt-add-repository "deb http://dl.google.com/linux/chrome/deb/ stable main"
+  - sudo sed -i s/deb-src.*google.*//g /etc/apt/sources.list
+  - wget -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
+  - sudo apt-get update -qq
+  - sudo apt-get install -y google-chrome-stable
+  - sudo chmod 1777 /dev/shm
```

Comment:
Hm, does this really require sudo? Have you tried adding google-chrome-stable to apt directly (maybe it's whitelisted by Travis) and just looking in the working path for chromedriver? Could we just download the Chrome .deb, `ar xv` & run it?

My concern with `sudo: required` (below) is that it forces us off of Travis's newer / preferred container infra: http://docs.travis-ci.com/user/workers/container-based-infrastructure/